packages/ui-server/routes/api/_entry.js

```diff
@@ -0,0 +1,70 @@
+const webpackEntryPoint = {};
+function forEachPromise(entries, fn) {
+	return entries.reduce((promise, prop) => {
+		const trimmedProp = prop.trim();
+
+		return promise.then((n) => {
+			if (n) {
+				Object.keys(n).forEach((val) => {
+					if (
+						n[val].charAt(0) !== "(" &&
+						n[val].charAt(0) !== "[" &&
+						!n[val].includes("function") &&
+						!n[val].includes("path") &&
+						!n[val].includes("process")
+					) {
+						n[val] = `\'${n[val].replace(/"|'/g, "").concat(".js")}\'`;
+					}
+					webpackEntryPoint[val] = n[val];
+				});
+			} else {
+				n = {};
+			}
+			return fn(trimmedProp);
+		});
+	}, Promise.resolve());
+}
+
+module.exports = function(questioner, answer) {
```

Comment:
@rishabh3112 looking first in the code, I didn't know what `answer` mean. After, reading the code I understood that is an answer for the question about multiple bundle. We could rename this for `usingMultipleBundles`, it will reduce the time to understand this part.

Reply:
Sure will make it once the port is complete.
Thanks for pointing out 😀

Reply:
JsDoc is added for this for now